### PR TITLE
Telco page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Build telco page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit)
- Check navbar links go to the correct URL (some URL don't have endpoints yet)
- Marketo form is being built in another issue so will not work yet.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4188

### CODE REVIEW

**Comment by ClementChaumel** [comment]
Looking good to me, what is the plan with this PR? Is it blocked until the missing pages and the Marketo form are implemented?

File: templates/telco/index.html [hunk 1]
```diff
@@ -43,836 +38,545 @@ <h1>
...
-      </p>
+    <div class="col-6">
+      <h2>Infrastructure for NFV</h2>
+      <h4>OpenStack engineered for best price–performance</h4>
+      <p>Designed to be economical in every way.</p>
+      <p>Smart operations, optimal architecture, better pricing.</p>
+      <p>All of this to ensure TCO reduction of your cloud, while expanding your budget for innovation.</p>
       <p>
-        <a  href="/engage/ubuntu-core-security-audit">Read the whitepaper: "Ubuntu Core: a cybersecurity analysis"&nbsp;&rsaquo;</a>
+        <a href="/vnf">Read more&nbsp;&rsaquo;</a>
```
Comment (ClementChaumel) [inline templates/telco/index.html:262]: Shouldn't this link be "/telco/openstack" instead?

**Comment by petesfrench** [comment]
@ClementChaumel For now if this is ready I will merge into the telco-bubble feature branch and when the form is ready include that. I have the logic in the page to access the form, so just need the form to exist.

File: templates/telco/index.html (hunk 1 above)
Comment (petesfrench) [reply templates/telco/index.html:262]: The vnf page is talking about openstack, or did you mean to change the URL?

**Comment by ClementChaumel** [comment]
> @ClementChaumel For now if this is ready I will merge into the telco-bubble feature branch and when the form is ready include that. I have the logic in the page to access the form, so just need the form to exist.

:+1:  Just be careful at the moment the PR is set to merge on master

File: templates/telco/index.html (hunk 1 above)
Comment (ClementChaumel) [reply templates/telco/index.html:262]: Yeah in the navigation you defined VNF like so 
```yaml
    - title: VNF
      path: /telco/openstack
```
so I just wondered if `/vnf` shouldn't be replaced with `/telco/openstack` here

**Comment by spencerbygraves** [comment]
Thanks, a couple of things from me please.

- Can you even up the logo sizes on the second logo strip please, to better match the first
- Here is the image asset for the Micro clouds section - https://assets.ubuntu.com/v1/1bb2f79a-telco-micro-clouds.svg
- I'll review the Full stack, one price section when it's updated

Thanks

**Comment by spencerbygraves** [comment]
The Full stack section should have individual logos rather than the image so we don't repeat the title, and can you match the text alignment to [the design](https://github.com/canonical-web-and-design/web-squad/issues/4187#issuecomment-874144815) please.

Thanks

**Review by carkod — APPROVED** [review]

**Pull request merged**